source/create.ts

```diff
@@ -39,7 +39,7 @@ type OptionsOfJSONResponseBody = Merge<Options, {isStream?: false; resolveBodyOn
 type OptionsOfBufferResponseBody = Merge<Options, {isStream?: false; resolveBodyOnly?: false; responseType: 'buffer'}>;
 type ResponseBodyOnly = {resolveBodyOnly: true};
 
-interface GotFunctions {
+export interface GotRequestMethod {
```

Comment:
Add a doc comment here describing what it is and how it might be useful to user.s